src/pages/oversikt/kandidatliste/Kandidatliste.tsx

```diff
@@ -16,13 +16,19 @@ interface Props {
 
 const Kandidatliste = ({ kandidater, onClickKandidat, antallValgteKriterier }: Props) => {
     const [visKandidater, toggleVisKandidater] = useState<boolean>(true);
+    const [counter, setCounter] = useState<number>(0);
 
     useLayoutEffect(() => {
-        toggleVisKandidater(false);
+        if (counter > 1) {
+            toggleVisKandidater(false);
 
-        setTimeout(() => {
-            toggleVisKandidater(true);
-        });
+            setTimeout(() => {
+                toggleVisKandidater(true);
+            });
+        }
+
+        setCounter(counter + 1);
+        // eslint-disable-next-line
```

Comment:
Hvorfor ikke legge til counter i dependencies, i stedet for å ignore warningen?

Reply:
Hm, ser at det ikke funker. Men burde nok fikse det underliggende problemet i stedet for ignore